metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -470,11 +471,10 @@ def get_views_for_dataset(
             BigqueryView(
                 name=table.table_name,
                 created=table.created,
-                last_altered=table.last_altered
-                if "last_altered" in table
-                else table.created,
+                last_altered=table.get("last_altered", table.created),
```

Comment:
When looking at the source code, I'm pretty sure that `conn.result()` emits `Row` objects, which implement `__getattr__` and `__getitem__` but not `__contains__`, so I don't think the old check worked.